package-lock.json

```diff
@@ -1,17438 +1,8 @@
 {
   "name": "ipfs-docs",
   "version": "2.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Why is package.json modified so heavily?

Reply:
I think that happened because I did a ```npm i``` instead of ```npm ci``` so the package-lock got overwritten

Reply:
@hsanjuan fixed it, sorry for the trouble.